core/src/epicli/data/common/ansible/playbooks/roles/helm/tasks/install-bin.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Download and unpack Helm binary
+  unarchive: 
+    src: "{{ specification.apache_epirepo_path }}/files/helm-{{ specification.version }}-linux-amd64.tar.gz"
+    dest: "/usr/local/bin"
+    extra_opts:
+      - --strip=1
+      - --wildcards
+      - '*/helm'
+    remote_src: yes
```

Comment:
I'd suggest using `true/false` instead of `yes/no`, as `yes/no` is specific to ansible and openssh, but `true/false` you can find everywhere. Just something to think about.